raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,335 @@
+            - assert: {from: 1, to: 2, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 0, balance: 0,  state: "opened"}
+      - parallel:
+          name: "A B C D make a deposit of 100 to their channels"
+          tasks:
+            - deposit: {from: 0, to: 1, total_deposit: 100}
+            - deposit: {from: 1, to: 0, total_deposit: 100}
+            - deposit: {from: 1, to: 2, total_deposit: 100}
+            - deposit: {from: 2, to: 1, total_deposit: 100}
+            - deposit: {from: 2, to: 3, total_deposit: 101}
+            - deposit: {from: 3, to: 2, total_deposit: 101}
+      - parallel:
+          name: "Checking if the deposit was successful"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 100, balance: 100, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100, balance: 100, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 100, balance: 100, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100, balance: 100, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 101, balance: 101, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101, balance: 101, state: "opened"}
+      - parallel:
+          name: "Deposit to a channel that does not exist"
+          tasks:
+            # Node tries to deposit to an unexisting channel
+            - deposit: {from: 4, to: 3, total_deposit: 20, expected_http_status: 409}
+            # Node tries to deposit less then the total deposit in two of his channels
+            - deposit: {from: 1, to: 0, total_deposit: 50, expected_http_status: 409}
+            - deposit: {from: 1, to: 2, total_deposit: 50, expected_http_status: 409}
+            # Opening a channel, but the channel already exists
+            - open_channel: {from: 1, to: 0, total_deposit: 100, expected_http_status: 409}
+      - serial:
+          name: "Give the nodes time to sync"
+          tasks:
+            - wait: 10
+      - parallel:
+          name: "Checking for the channel state to be unchanged"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 100, balance: 100, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100, balance: 100, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 100, balance: 100, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100, balance: 100, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 101, balance: 101, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101, balance: 101, state: "opened"}
+      - serial:
+          name: "C tries different cases"
```

Comment:
We have agreed to use numbers instead of letters to describe the nodes in general, since that's what the scenario player uses internally and this makes it easier to reason about.